UPGRADING.rst

```diff
@@ -10,6 +10,32 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#3497 Fix: API Gateway log is missing important fields
+======================================================
+
+Upgrades to the logs format for API Gateway require recreating
+the REST API infrastructure and its dependencies. Run::
+
+    (cd terraform && terraform destroy \
+    -target=aws_api_gateway_domain_name.service_0 \
+    -target=aws_api_gateway_method_settings.service_1 && terraform destroy \
+    -target=aws_api_gateway_method_settings.service_0 && terraform destroy \
+    -target=module.chalice_indexer.aws_api_gateway_rest_api.rest_api \
+    -target=module.chalice_service.aws_api_gateway_rest_api.rest_api) \
+    && make deploy
+
+Troubleshooting:
+Some errors regarding log groups for API Gateway might occur while performing
+this upgrade. The last steps should be performed in quick succession.
+
+1. Run ``make deploy`` but don't type "yes".
```

Comment:
This is a little confusing. Is this the same `make deploy` included in the `(cd terraform && terraform destroy ...` command above, or should this `make deploy` be run after the above command completes?

Reply:
Yeah, is meant to be run after the first command if errors are encountered.